### PR TITLE
Support `async` and `throws` on property/subscript getters.

### PR DESCRIPTION
**Pull request by allevato**

### CODE REVIEW

**Comment by allevato** [comment]
cc @dylansturg

**Comment by jaredh159** [comment]
thanks to the maintainers and contributors for the great project, really appreciate it!

Is anything holding this PR up?  I currently can't write throwing getters without swift-format deleting them. 😢   Would be ver grateful for a new release containing the fix.

**Review by dylansturg — APPROVED** [review]

**Pull request merged**
